### DISCUSSION TITLE
Multiple windows display each other's different screens in the same session

### DISCUSSION THREAD

**Discussion by itbencn** [rustdesk/rustdesk#4910] [Feature Request]
1.In the same remote connection session, the other party has multiple screens, and I also have multiple screens myself. It is completely possible to use multiple windows to independently display different screens, improving work efficiency.

2.Even when the other party only has one screen, they can directly extend a remote virtual secondary screen for their own use, which may be difficult and can be ignored.

**Comment by jasonmacer** [comment]
> 1.In the same remote connection session, the other party has multiple screens, and I also have multiple screens myself. It is completely possible to use multiple windows to independently display different screens, improving work efficiency.
> 
> 2.Even when the other party only has one screen, they can directly extend a remote virtual secondary screen for their own use, which may be difficult and can be ignored.

Definitely this right here!  I use multiple monitors and almost all of our end users do as well. it would be nice to be able to open either multiple sessions or multiple windows for different displays!

**Comment by itbencn** [comment]
@rustdesk

**Comment by davidshq** [comment]
This would be a huge usability boost for me!

The other thing that could be done is how RDP handles it when the client has only a single monitor but the host system has multiple monitors - it seems to use a virtual monitor and collapse all of the windows open on the multiple monitors onto a single monitor. Then there is no need to tab between monitors.
